### ISSUE TITLE
dynamo batchGet error

### ISSUE THREAD

**Issue by joeb37** [CS599-MEDLINE/pubminer-ui-investigation#45]
From the UI search for "crossword."  The page indicates that 20 of 22 records are displayed. Press 'More' to load the next batch.  The following displays in the server console:
```
dynamo batchGet error: ValidationException: 1 validation error detected: Value '[]' at 'requestItems.demographics.member.keys' failed to satisfy constraint: Member must have length greater than or equal to 1
(node:27922) UnhandledPromiseRejectionWarning: TypeError: Cannot read property '9061440' of undefined
    at summaryResults.result.uids.map.resultItem (/Users/Joe/Developer/cscie599/pmc-oa-test/search.js:104:48)
    at Array.map (<anonymous>)
    at demoSvc.getDemographicDetailsForIds.then.demoDetails (/Users/Joe/Developer/cscie599/pmc-oa-test/search.js:102:30)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:160:7)
(node:27922) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:27922) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```
Something is causing this operation to be retried, as this message repeats in the console every 30 seconds or so.

The problem might be further upstream.  batchGet could be receiving bad data.

**Comment by freshottr** [comment]
@joeb37 superficially it looks like an empty set of keys is being passed into the `batchGet` method, which likely means that the summary results didn't have any PMC IDs associated. I should be able to fix this tonight or tomorrow.

**Comment by freshottr** [comment]
Addressed by #46
